README.md

```diff
@@ -33,7 +34,7 @@ To get a local copy up and running follow these simple example steps.
 
 - Open your Terminal/Powershell or any command line interface you use, then navigate to your created above directory or the directory you'll like to clone this project directory
 - Type `git clone https://github.com/LIBERCOSOFT/honky-tonk-music-fest.git` if your're using https with git or `git clone git@github.com:LIBERCOSOFT/honky-tonk-music-fest.git` if your preferred is ssh and hit enter
-- Navigate to the `libercosoft-portfolio` directory on your local computer in your above chosen directory and open the `index.html` file
+- Navigate to the `music-festival` directory on your local computer in your above chosen directory and open the `index.html` file
```

Comment:
- [ ] it would be better if you give credit to the author of the design as in the requirements